code/go/0chain.net/chaincore/chain/protocol_block.go

```diff
@@ -198,18 +198,19 @@ func (c *Chain) reachedNotarization(round int64, hash string,
 		}
 	}
 	if c.ThresholdByStake() > 0 {
-		verifiersStake := 0
+		verifiersStake := uint64(0)
 		for _, ticket := range bvt {
-			verifiersStake, err = maths.SafeAddInt(verifiersStake, c.getMiningStake(ticket.VerifierID))
+			verifiersStake, err = maths.SafeAddUInt64(verifiersStake, c.getMiningStake(ticket.VerifierID))
 			if err != nil {
 				logging.Logger.Error("reached_notarization", zap.Error(err))
 				return false
 			}
 		}
-		if verifiersStake < c.ThresholdByStake() {
+
+		if c.ThresholdByStake() > 0 && verifiersStake < uint64(c.ThresholdByStake()) {
```

Comment:
c.ThresholdByStake() >  0 is already checked above, so we don't need to do it again